config.go

```diff
@@ -132,17 +132,24 @@ func (c *Config) ParseKeys() ([]common.Address, []*ecdsa.PrivateKey, error) {
 	return addresses, privKeys, nil
 }
 
+// IsSimulatedNetwork returns true if the network is simulated (i.e. Geth or Anvil)
 func (c *Config) IsSimulatedNetwork() bool {
 	networkName := strings.ToLower(c.Network.Name)
 	return networkName == strings.ToLower(GETH) || networkName == strings.ToLower(ANVIL)
 }
 
+// GenerateContractMapFileName generates a file name for the contract map
 func (c *Config) GenerateContractMapFileName() string {
 	networkName := strings.ToLower(c.Network.Name)
 	now := time.Now().Format("2006-01-02-15-04-05")
 	return fmt.Sprintf(ContractMapFilePattern, networkName, now)
 }
 
+// ShoulSaveDeployedContractMap returns true if the contract map should be saved (i.e. not a simulated network and functionality is enabled)
+func (c *Config) ShoulSaveDeployedContractMap() bool {
```

Comment:
We do not include some functionality for `simulated` networks but we use it all the time with static/real networks, for example `keyfile.toml` is required if we are not in ephemeral mode. Maybe we can use `ephemeral` flag for this case too? 
It doesn't make sense to save any data for ephemeral networks since we can always run it fast from block zero, but it always makes sense to have as much info as possible with a static network, wdyt? @Tofel